src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -31,29 +31,34 @@
 
 
 DATA_SAVE_ID  = wx.NewId()
-
+# NOTE: for reference, only comment out unwanted icons
+CUSTOMIZED_TOOLITEMS = (
+        ('Home', 'Reset original view', 'home', 'home'),
+        ('Back', 'Back to  previous view', 'back', 'back'),
+        ('Forward', 'Forward to next view', 'forward', 'forward'),
+        (None, None, None, None),
+        ('Pan', 'Pan axes with left mouse, zoom with right', 'move', 'pan'),
+        ('Zoom', 'Zoom to rectangle', 'zoom_to_rect', 'zoom'),
+        #('Subplots', 'Configure subplots', 'subplots', 'configure_subplots'),
+        (None, None, None, None),
+        #('Save', 'Save the figure', 'filesave', 'save_figure')
+        )
 
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var before init
+        NavToolbar.toolitems = CUSTOMIZED_TOOLITEMS
```

Comment:
Not good - this modifies the base class.  Please leave it alone and just define `tooltimes` above as an attribute of the ExtendedToolbar class:
```python
toolitems = tuple(ti for ti in NavToolbar.toolitems if ti[0] != 'Subplots')
```